atlasdb-config/src/main/java/com/palantir/atlasdb/config/AtlasDbConfig.java

```diff
@@ -223,6 +223,15 @@ public int getDefaultLockTimeoutSeconds() {
         return AtlasDbConstants.DEFAULT_LOCK_TIMEOUT_SECONDS;
     }
 
+    /**
+     * Returns a configuration for this timestamp client, indicating whether timestamp batching
+     * should be used, and if so, the minimum time interval between batches.
+     */
+    @Value.Default
+    public TimestampClientConfig timestampClient() {
```

Comment:
I've added this. Note that this incurs two volatile reads on each timestamp request (see internal live reloading library for details), but given that I expect much of the time to be spent in/around actually waiting for batches/responses the impact is likely to be acceptable.